pkg/metrics/reconciliation_status.go

```diff
@@ -0,0 +1,73 @@
+package metrics
+
+import (
+	"fmt"
+
+	"github.com/kyma-incubator/reconciler/pkg/cluster"
+	"github.com/kyma-incubator/reconciler/pkg/model"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	prometheusSubsystem = "reconciler"
+
+	statusError            float64 = 0
```

Comment:
Maybe we can integrate this mapping into the status-struct we are using in our model. If we change laters our statuses, we would not have to update the metrics as well.